### PR TITLE
Added 3D auto and vanes L/R

### PR DESCRIPTION
**Pull request by glsf91**
Added 3D auto and vanes L/R. You have to enable this.

### CODE REVIEW

**Comment by glsf91** [comment]
@absalom-muc Please do not merge it now.
I first want to ask a few people to test. When done I will merge it into master

File: SW-Configuration.md
Comment (RudolfRendier) [inline SW-Configuration.md:190]:
```suggestion
//#define USE_EXTENDED_FRAME_SIZE true                // uncomment if you want to use de extended frame size (33) which is used by WF-RAC module
```

File: src/MHI-AC-Ctrl-core.cpp
Comment (RudolfRendier) [inline src/MHI-AC-Ctrl-core.cpp:141]:
```suggestion
  if (frameSize == 33)
```

File: src/MHI-AC-Ctrl-core.cpp
Comment (RudolfRendier) [inline src/MHI-AC-Ctrl-core.cpp:223]:
```suggestion
  if (frameSize == 33) { // Only for framesize 33 (WF-RAC)
```

File: src/MHI-AC-Ctrl-core.cpp
Comment (RudolfRendier) [inline src/MHI-AC-Ctrl-core.cpp:270]:
```suggestion
  if (frameSize == 33) { // Only for framesize 33 (WF-RAC)
```

File: src/MHI-AC-Ctrl-core.cpp
Comment (RudolfRendier) [inline src/MHI-AC-Ctrl-core.cpp:272]:
```suggestion
    if (MOSI_frame[CBL2] != lowByte(checksum)) 
```

File: SW-Configuration.md
Comment (RudolfRendier) [inline SW-Configuration.md:188]:
```suggestion
But when you uncomment the following line, the frame size is extended to 33 bytes (like the WF-RAC module). This will make it possible to use 3D auto and vanes L/R.
```

File: src/support.h
Comment (RudolfRendier) [inline src/support.h:51]:
```suggestion
//#define USE_EXTENDED_FRAME_SIZE true                // uncomment if you want to use de extended frame size (33) which is used by WF-RAC module
```

**Review by RudolfRendier — CHANGES_REQUESTED** [review]
Thanks for this addition.  Noticed some minor cosmetic issue
Looking forward to this.

File: src/MHI-AC-Ctrl-core.h
Comment (RudolfRendier) [inline src/MHI-AC-Ctrl-core.h:188]: Copy paste error from the line above it

**Comment by glsf91** [comment]
> Thanks for this addition. Noticed some minor cosmetic issues. Looking forward to this.

Thanks. I will take over this changes when no other changes are needed anymore before I will do a merge

**Pull request merged**
